Mathlib/GroupTheory/SpecificGroups/Alternating.lean

```diff
@@ -105,9 +105,11 @@ theorem alternatingGroup.index_eq_two [Nontrivial α] :
   simp_rw [mem_top, Nat.card_eq_fintype_card]; rfl
 
 @[nontriviality]
-theorem index_eq_one [Subsingleton α] : (alternatingGroup α).index = 1 := by
+theorem alternatingGroup.index_eq_one [Subsingleton α] : (alternatingGroup α).index = 1 := by
   rw [Subgroup.index_eq_one]; apply Subsingleton.elim
 
+@[deprecated (since := "2025-02-01")] alias index.eq_one := alternatingGroup.index_eq_one
+
```

Comment:
I don't think you need to deprecate it because it's only one day old.
```suggestion
```
(Also the name is `index_eq_one` not `index.eq_one`.)